src/node.cpp

```diff
@@ -142,4 +148,35 @@ std::vector<session_key_t> Node::getReadySessions() const {
     return std::move(readySessions);
 }
 
+Status Node::demultiplyOutputs(SessionResults& nodeSessionOutputs) {
+    auto& [metadata, blobMap] = nodeSessionOutputs.begin()->second;
+    auto& [session, blob] = *blobMap.begin();
```

Comment:
```suggestion
    auto& [blobName, blob] = *blobMap.begin();
```

blobName is not used anyway but naming it session is confusing.